app/controllers/api/v1/cuisines_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::CuisinesController < ApplicationController
+  def index
+    render json: Cuisine.all
+  end
+
+  def show
+    render json: Cuisine.find(params[:id])
+  end
```

Comment:
This PR doesn't seem to use the `show` endpoint so please remove this and save for a later PR!